libs/deps/mas/merch-card.js

```diff
@@ -975,7 +975,12 @@ merch-card[variant="catalog"] [slot="footer"] .spectrum-Link--primary {
   }
 
   merch-card[variant="mini-compare-chart"] .footer-row-cell-description a {
-    color: var(--color-accent);
+    color: var(--link-color);
+  }
+
+  merch-card[variant="mini-compare-chart"] .footer-row-cell-description a:focus,
+  merch-card[variant="mini-compare-chart"] .footer-row-cell-description a:hover {
+    color: var(--link-hover-color);
```

Comment:
Did you sync with someone from the merch team on how to best approach changes in MAS files? I remember changes being needed in one file and then having to run a build script to propagate them. @adobecom/tacocat might have guidance on this. And maybe some documentation we can use in the future if changes to these files are required?

Reply:
This file should not be patched. Likely you want to patch global.css.js and `npm run build` in libs/features/mas folder